test/conformance/CMakeLists.txt

```diff
@@ -8,9 +8,18 @@ set(UR_CONFORMANCE_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 function(add_test_adapter name adapter)
     set(TEST_TARGET_NAME test-${name})
     set(TEST_NAME ${name}-${adapter})
+
     add_test(NAME ${TEST_NAME}
-        COMMAND ${TEST_TARGET_NAME}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+        COMMAND ${CMAKE_COMMAND}
+        -D TEST_FILE=${Python3_EXECUTABLE}
```

Comment:
Is this solution like a temporary one? Or should we maintain this list of "failing" tests forever, even if all of them are green?

Perhaps we should mention somewhere (at least in a comment here) what's the purpose of this extra matching here...

Reply:
Ideally all adapters would have 100% pass rate, but that's not the case at the moment unfortunately. And it's going to take some time for everything to get fixed. So yes, this is permanent until such time that we fix everything and agree to never break CTS again, which might be a while.

Definitely should be documented somewhere.

Reply:
Done.

Reply:
Thanks!